contrib/system_containers/centos/Dockerfile

```diff
@@ -11,7 +11,8 @@ LABEL com.redhat.component="cri-o" \
       maintainer="Yu Qi Zhang <jzehrarnyg@gmail.com>" \
       atomic.type="system"
 
-RUN yum-config-manager --nogpgcheck --add-repo https://cbs.centos.org/repos/virt7-container-common-candidate/x86_64/os/ && \
+RUN yum-config-manager --nogpgcheck --add-repo https://buildlogs.centos.org/centos/7/virt/x86_64/container && \
+    yum-config-manager yum-config-manager --nogpgcheck --add-repo https://buildlogs.centos.org/centos/7/paas/x86_64/openshift-origin39/ && \
```

Comment:
There's `yum-config-manager yum-config-manager` back-to-back. Just one's needed :) . But the rest LGTM